frontend/projects/upgrade/src/app/features/dashboard/home/components/metrics/metrics.component.ts

```diff
@@ -519,7 +519,6 @@ export class MonitoredMetricsComponent implements OnInit, OnChanges, OnDestroy {
     if (this.isContextChanged) {
       this.isContextChanged = false;
       this.queries.clear();
-      this.queries.push(this.addMetric());
```

Comment:
same as other Q, what does removing this do?

Reply:
I'm not 100% sure if I did it correctly but removing the `this.queries.push(this.addMetric());` removed the default row in the Metrics table. The same changes for the other steps. I need confirmation from @ppratikcr7 if these changes are the correct way to remove the default rows.

Reply:
@zackcl Correct

Reply:
@ppratikcr7 Thank you!